src/main/chatBot/BotLogic.java

```diff
@@ -0,0 +1,50 @@
+package main.chatBot;
+
+public class BotLogic {
+
+    private enum State {
+        START,
+        PROCEED,
+        WAITING,
+        STOP
+    }
+
+    private String[] feels = {"excellent ", "not bad ", "awesome ", "very bad ", "interesting", "wonderful", "annoyed"};
+    private String[] questions = {" How are you today?", " What's happening?"};
+
+    private State state = State.START;
+
+    public String process(String input) {
+        String output;
+
+        switch (state) {
+            case START: {
+                output = "Hi, I am a super Bot! I am a psychologist! Do you want to tell me about your problems?(yes/no)";
+                state = State.PROCEED;
+                break;
+            }
+            case PROCEED: {
+                if (input.equalsIgnoreCase("yes")) {
+                    output = feels[0].concat(questions[0]).concat(questions[1]);
```

Comment:
badPractice, wrap this part in a particular method.

Reply:
> 
I mean whole case Proceed
>

>

Reply:
understood, I will do it now

Reply:
fixed it

Reply:
can I merge it?